front/components/assistant/conversation/content_creation/ShareContentCreationFilePopover.tsx

```diff
@@ -128,45 +126,58 @@ export function ShareContentCreationFilePopover({
   const [isOpen, setIsOpen] = React.useState(false);
   const [isCopied, copyToClipboard] = useCopyToClipboard();
   const [isUpdatingShare, setIsUpdatingShare] = React.useState(false);
-  const [selectedScope, setSelectedScope] = React.useState<FileShareScope>(
-    "conversation_participants"
-  );
+  const [selectedOption, setSelectedOption] =
+    React.useState<SharingOption>(PRIVATE_OPTION);
 
   const isPublicSharingForbidden =
     owner.metadata?.allowContentCreationFileSharing === false;
 
+  const isPublished = selectedOption.value !== "none";
+
+  const options = getSharingOptions(owner.name);
+
   const { doShare, fileShare, isFileShareLoading, isFileShareError } =
     useShareContentCreationFile({
       fileId,
       owner,
     });
 
+  const shareURL = fileShare?.shareUrl ?? "";
+
   // Sync selectedScope with current fileShare data.
   React.useEffect(() => {
     if (!isFileShareLoading && !isFileShareError && fileShare) {
-      setSelectedScope(fileShare.scope);
+      const selectedOption = options.find(
+        (option) => option.value === fileShare.scope
+      );
+
+      if (selectedOption) {
+        setSelectedOption(selectedOption);
+      }
     }
-  }, [fileShare, isFileShareLoading, isFileShareError]);
+  }, [fileShare, isFileShareLoading, isFileShareError, options]);
+
+  const onPublish = async () => {
+    await handleChangeFileShare("workspace");
+  };
+
+  const placeholderUrl = `${window.location.origin}/w/${owner.sId}/assistant/conversation/share`;
 
   const handleChangeFileShare = async (shareScope: FileShareScope) => {
     setIsUpdatingShare(true);
     try {
       await doShare(shareScope);
-      setSelectedScope(shareScope);
```

Comment:
we don't really need this since it's set in `handleScopeChange`, and also the state gets synced in useEffect